index.html

```diff
@@ -1,39 +1,240 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
-    <title>Document</title>
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="http://fonts.googleapis.com/css?family=Roboto"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <title>Portfolio Kazaneza</title>
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <label class="logo"><a href="">KAZANEZA GENTIL</a></label>
+        <a href="#">
+          <img
+            class="navImage"
+            src="./Images/ic_menu.svg"
```

Comment:
- [ Optional] It's not a part of this milestone but the hamburger menu is not displaying, you have a typo in the image link. You can consider changing ` src="./Images/ic_menu.svg"` to ` src="./images/ic_menu.svg"`.